tests/test_bs.py

```diff
@@ -254,6 +261,54 @@ def fin():
     request.addfinalizer(fin)
     return _setup_clients_in_namespace(testconfig['namespace'])
 
+
+def add_single_client(deployment_id, container_specs):
+
+    resp = pod.create_pod(CLIENT_POD_FILE,
+                          testconfig['namespace'],
+                          deployment_id=deployment_id,
+                          container_specs=container_specs)
+
+    client_name = resp.metadata.name
+    print("Add new client: {0}".format(client_name))
+    return client_name
+
+
+@pytest.fixture()
+def add_client(request, setup_oracle, setup_poet, setup_bootstrap, setup_clients):
```

Comment:
add comment about this not being a functional fixture and shouldn't be used in our tests

Reply:
ok